csaf_2.1/prose/edit/src/distributing.md

```diff
@@ -560,14 +560,16 @@ Each such folder MUST at least:
 
 ### Requirement 24: HTTP User-Agent
 
-Access to the files provided, both CSAF metadata and documents, MUST NOT be restricted by the value of HTTP User-Agent.
+Access to the CSAF related files and directories provided, for both metadata and documents, MUST be allowed independent of the
+value of HTTP User-Agent.
 
 > Limit the value of HTTP User-Agents to a certain set would hinder adoption of tools retrieving the files.
 
 The only exception is that the temporary blocking of certain HTTP User-Agents is allowed to mitigate an ongoing security incident
-(e.g. a DoS attack on the web server serving the CSAF files). However, it MUST be checked if a less severe measure with a similar
-effect could be used. White-listing CSAF related files and directories MUST be considered. The temporary blocking MUST be removed
-as soon as possible, at latest two weeks after the security incident process was completed.
+(e.g. a DoS attack on the web server serving the CSAF files).
+However, it MUST be checked if a less severe measure with a similar effect could be used.
```

Comment:
The rewrites look good to me, but I think these three MUSTs shall become SHOULDs:

>However, a less severe measure with a similar effect SHOULD be used.
>CSAF related files and directories SHOULD be exempted from temporary blocking.
>The temporary blocking SHOULD be removed as soon as possible, at latest two weeks after the security incident process was completed.

I reordered and simplified the sentences as well as implicitly fixed a nit (`a the`).

Reply:
Hmm, I think I like the MUST better (especially thinking of my service provider) as it enforces (potentially with a documentation part, at least that is what I would require) to:
1. See whether less severe measures could be used.
2. Consider exempts
3. Remove the blocking at some point

I guess that is something the TC needs to decide what the best option is.

Reply:
@santosomar, @justmurphy, @mprpic: Looking for input from native speakers to find the best wording. Please also state your opinion whether the "SHOULD" option or the "MUST consider" (which allows to easily request reasoning as consumer) would be preferred.

Reply:
In my understanding SHOULD already means "there have to be compelling reasons not to do as generally this is the way to go).

Reply:
The wording has been changed to SHOULD as discussed in today's TC meeting.